docs/sample_config.yaml

```diff
@@ -142,6 +142,17 @@ presence:
 #
 #limit_profile_requests_to_users_who_share_rooms: true
 
+# User Directory Updates
+#
+# If specified, a worker with that name will be the only one able to update
+# the user directory. Important for querying shared rooms. If the worker is
+# specified, but not running, the user directory may become outdated.
+#
+# Defaults to null, meaning the main process will handle this.
+#
+# Set to false to make sure no process will handle the user_directory.
+#worker_to_update_user_directory: null
```

Comment:
please have a look at https://github.com/matrix-org/synapse/blob/develop/docs/code_style.md#configuration-file-format and ensure this matches.

Reply:
It should probably be in the `workers` section?

Reply:
and maybe it should be called `update_user_directory_on` for consistency with `run_background_tasks_on` ?

Reply:
(should it default to the same value as `run_background_tasks_on`?)

Reply:
I've addressed all three first issues, but the last issue is still unresolved, i'm leaving this thread open for discussion on that.